cluster-scope/base/rbac.authorization.k8s.io/clusterroles/curator-operator-manager-role/clusterrole.yaml

```diff
@@ -0,0 +1,148 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: curator-operator-manager-role
+rules:
+- apiGroups:
+  - batch
+  resources:
+  - cronjobs
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - batch
+  resources:
+  - cronjobs/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - batch
+  resources:
+  - cronjobs/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - batch
+  resources:
+  - jobs/status
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  verbs:
+  - create
+  - delete
```

Comment:
@skanthed why does this operator need write permissions to all pvc's on the cluster?

Reply:
@HumairAK Most of the RBAC are auto-generated when we are building an operator using operator-sdk. We have not written any logic that requires access to other PVCs. Even if we don't have access to other PVCs on the cluster, it should be good.

I can try changing it and test it on the curator cluster. I will update the PR accordingly.